### ISSUE TITLE
Integrate EcoStruxure Panel Server

### ISSUE THREAD

**Issue by brahmlategan** [Breina/PowerTagGateway#3]
Hi there, not sure if you know about it but the PowerTag Link is now obsolete and replaced by Schneider's PanelServer range

**Comment by Breina** [comment]
Thanks for the heads up! I wasn't aware of that yet.

The panel servers seem very powerful, though are more than double the price of the PowerTag Link. Doesn't seem like the PanelServers are intended for home use.

Gonna ask on EcoStruxure what's planned for this...

For my own reference; https://www.se.com/ww/en/product-range/40739468-ecostruxure-panel-server/?parent-subcategory-id=4160&filter=business-4-low-voltage-products-and-systems

**Comment by Breina** [comment]
@brahmlategan Looks like the PAS400 is in the works, which ought to be similar to PowerTag Link. I'll use this Issue to track integration with Panel Server API. The repo will need a rename then though.

**Comment by brahmlategan** [comment]
Yeah the Panelserver is a replacement for quite a few SE products. Just be aware that not all functionality is built into the Panelservers off the shelf, SE is releasing these piecemeal as firmware upgrades.

PAS400 and PAS600 is available, PAS800 have just been released I think. Last time I checked the WiFi component wasn't yet added to the firmware, but Zigbee is.

Also, something else you might want on your radar is the newly released Acti9 Active range - miniature circuit breakers with AFD, RCD, energy monitoring and Zigbee comms built in - https://www.se.com/uk/en/product-range/45435131-acti9-active/
